tests/PHPUnit/Framework/Constraint/HttpResponseText.php

```diff
@@ -7,7 +7,9 @@
  */
 namespace Piwik\Tests\Framework\Constraint;
 
-class HttpResponseText extends \PHPUnit_Framework_Constraint
+use PHPUnit\Framework\TestCase;
+
+class HttpResponseText extends Constraint
```

Comment:
Tests fail here, as `Constraint` class can't be found. Guess you need to adjust the `use` statement

Reply:
Thank you, I didn't get this one. I'll amend the commit and change it.